packages/gatsby-theme-docs/src/layouts/internals/layout-page-navigation.js

```diff
@@ -135,7 +135,12 @@ const LayoutPageNavigation = (props) => {
       <SpacingsStack scale="m">
         <PageTitleLink href="#top">
           <SpacingsInline scale="s">
-            <div>
+            <div
+              css={css`
+                margin-top: ${designSystem.dimensions.heights
+                  .arrowUpIconMarginTop};
+              `}
+            >
```

Comment:
Instead of using negative margings, I'd rather add some padding to the element on the right. For example 1 or 2 pixels.

<img width="576" alt="image" src="https://user-images.githubusercontent.com/1110551/106252346-7532d400-6216-11eb-9993-566c23a122de.png">